taxi/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse
+from taxi.models import Driver, Manufacturer, Car
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code. Also, separate different types of imports in this order: standard library, external library, custom modules.

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)